plyer/platforms/linux/wifi.py

```diff
@@ -16,8 +16,31 @@
 
 
 class LinuxWifi(Wifi):
+    def __init__(self):
+        self.ifname = self.getIfname()
+        self.nmcliVersion = self.getNmcliVersion()
+
     names = {}
 
+    def getIfname(self):
+        '''
+        Return wifi interface name if available else return "wlan0'.
+        '''
+        com = Popen(["nmcli", "-t", "device"], stdout=PIPE)
+        com = com.communicate()[0].decode().split("\n")
+        for i in com:
+            if("wifi" in i):
+                return i.split(":")[0]
+        return 'wlan0'
+
+    def getNmcliVersion(self):
+        '''
+        Return nmcli version in Int to perform operation accordingly.
+        '''
+        com = Popen(["nmcli", "--version"], stdout=PIPE)
+        com = com.communicate()[0].decode("utf-8").strip()
+        return int(com.split(" ")[-1].replace(".", ""))
```

Comment:
In case you get a stupid value back from the process or you don't get *any* value back for whatever reason, you'll end up with `int('')` or `int('bad')` which will raise an error instead of continuing in the workflow normally where the commands for disconnect might actually work nevertheless. This must not fail. :)